queries.sql

```diff
@@ -5,4 +5,21 @@ SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name != 'Gabumon';
-SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+begin
+UPDATE animals SET specie = 'unspecified';
+SELECT * FROM animals;
+rollback
```

Comment:
- [ ] You missed the below queries to add to your queries file and also make sure to dd screenshots with the results of your queries in the PR description.

![image](https://user-images.githubusercontent.com/83803414/230101752-18bb233a-84aa-4438-8db1-c4d4fef2ac27.png)